packages/builders/__tests__/components/selectMenu.test.ts

```diff
@@ -176,5 +186,11 @@ describe('Select Menu Components', () => {
 			).toEqual(selectMenuData);
 			expect(new StringSelectMenuOptionBuilder(selectMenuOptionData).toJSON()).toEqual(selectMenuOptionData);
 		});
+
+		test('GIVEN a StringSelectMenuBuilder using StringSelectMenuBuilder#spliceOptions works', () => {
+			expect(makeStringSelectMenuWithOptions().spliceOptions(0, 1).options.length).toBe(2);
+			expect(makeStringSelectMenuWithOptions().spliceOptions(0, 1, selectMenuOptionData).options.length).toBe(3);
+			expect(makeStringSelectMenuWithOptions().spliceOptions(0, 3, selectMenuOptionData).options.length).toBe(1);
```

Comment:
We shouldn't test comparing lengths, but comparing if the output is the expected result, using `isStrictEquals`.